apps/console/app/styles/global.css

```
@@ -0,0 +1,12 @@
+/* Global cross-fade duration */
+::view-transition-old(root),
```

Biome (2.1.2)

[error] Unknown type selector is not allowed.

See MDN web docs for more details.
Consider replacing the unknown type selector with valid one.

(lint/correctness/noUnknownTypeSelector)